Build fails on Linux w/ Clang

Building with Clang on Arch Linux eventually fails with the following errors:

```
/home/marisa/.cache/paru/clone/rpcs3-git/src/rpcs3/rpcs3/Input/gui_pad_thread.cpp:58:13: error: use of undeclared identifier 'close'
   58 |                 int res = close(m_uinput_fd);
      |                           ^
/home/marisa/.cache/paru/clone/rpcs3-git/src/rpcs3/rpcs3/Input/gui_pad_thread.cpp:569:12: error: use of undeclared identifier 'write'; did you mean '_IO_cookie_io_functions_t::write'?
  569 |         int res = write(m_uinput_fd, &ie, sizeof(ie));
      |                   ^~~~~
      |                   _IO_cookie_io_functions_t::write
/usr/include/bits/types/cookie_io_functions_t.h:58:28: note: '_IO_cookie_io_functions_t::write' declared here
   58 |   cookie_write_function_t *write;       /* Write bytes.  */
      |                            ^
/home/marisa/.cache/paru/clone/rpcs3-git/src/rpcs3/rpcs3/Input/gui_pad_thread.cpp:569:12: error: invalid use of non-static data member 'write'
  569 |         int res = write(m_uinput_fd, &ie, sizeof(ie));
      |                   ^~~~~
3 errors generated.
make[2]: *** [rpcs3/CMakeFiles/rpcs3.dir/build.make:293: rpcs3/CMakeFiles/rpcs3.dir/Input/gui_pad_thread.cpp.o] Error 1
```

Easiest guess here is that `unistd.h` is not explicitly included, and while GCC *(13.2.1)* will apparently just carry on, this doesn't fly for Clang *(17.0.6)*.

--- a/Utilities/JITASM.cpp
+++ b/Utilities/JITASM.cpp
@@ -19,6 +19,7 @@ LOG_CHANNEL(jit_log, "JIT");
 void jit_announce(uptr func, usz size, std::string_view name)
 {
 #ifdef __linux__
+#if 0
 	static const struct tmp_perf_map
 	{
 		std::string name{fmt::format("/tmp/perf-%d.map", getpid())};
@@ -44,6 +45,7 @@ void jit_announce(uptr func, usz size, std::string_view name)
 		fs::remove_file(s_map.name);
 		return;
 	}
+#endif
 #endif
 
 	if (!size)

--- a/rpcs3/Input/gui_pad_thread.cpp
+++ b/rpcs3/Input/gui_pad_thread.cpp
@@ -22,6 +22,7 @@
 #ifdef __linux__
 #include <linux/uinput.h>
 #include <fcntl.h>
+#include <unistd.h>
 #define CHECK_IOCTRL_RET(res) if (res == -1) { gui_log.error("gui_pad_thread: ioctl failed (errno=%d=%s)", res, strerror(errno)); }
 #elif defined(__APPLE__)
 #pragma GCC diagnostic push

